### PR TITLE
Notify failing CD pipelines in Slack

### PR DESCRIPTION
**Pull request by carrodher**
### Description of the change

Use the same approach as the one used in bitnami/charts to notify failing CD pipelines in Slack

### CODE REVIEW

File: .github/workflows/cd-pipeline.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
-name: '[CI/CD] CD Pipeline Run'
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+name: '[CI/CD] CD Pipelin'
```
Comment (FraPazGal) [inline .github/workflows/cd-pipeline.yml:4]:
```suggestion
name: '[CI/CD] CD Pipeline'
```

File: .github/workflows/cd-pipeline.yml [hunk 2]
```diff
@@ -87,3 +90,45 @@ jobs:
...
+      - vib-publish
+      - update-index
+    if: always()
+    name: Notify unsuccessful CD run
+    steps:
+      - name: Notify in Slack channel
+        if: ${{ needs.vib-publish.result != 'success' || needs.update-index.result != 'success' }}
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: ${{ secrets.CD_SLACK_CHANNEL_ID }}
```
Comment (FraPazGal) [inline .github/workflows/cd-pipeline.yml:107]: Both this envar and `CD_SLACK_BOT_TOKEN` are set at the repo level for `bitnami/charts`, so we may need to add them in this repo still (I can't access the settings and confirm).

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @carrodher! I added a couple of minor comments, please check them whenever you can.

File: .github/workflows/cd-pipeline.yml (hunk 2 above)
Comment (carrodher) [reply .github/workflows/cd-pipeline.yml:107]: Thanks for your review, both should be added, see 
Screenshot 2023-07-21 at 17 49 02

**Pull request merged**